internetarchive/config.py

```diff
@@ -157,14 +158,14 @@ def parse_config_file(config_file=None):
     return (config_file, is_xdg, config)
 
 
-def get_config(config=None, config_file=None):
-    _config = {} if not config else config
+def get_config(config=None, config_file=None) -> dict:
```

Comment:
`config: mapping | None = None`. For `config_file`the same as for the comment above applies. It would currently be `string | None`, but we most likely want it to be a path in the future so we can skip that one for now.

Reply:
### Try `Mapping | None` ...
internetarchive/config.py:166: error: Incompatible return value type (got "Mapping[Any, Any]", expected "Dict[Any, Any]")
internetarchive/config.py:169: error: Item "Mapping[Any, Any]" of "Optional[Mapping[Any, Any]]" has no attribute "sections"
internetarchive/config.py:169: error: Item "None" of "Optional[Mapping[Any, Any]]" has no attribute "sections"
internetarchive/config.py:171: error: Item "None" of "Optional[Mapping[Any, Any]]" has no attribute "items"
internetarchive/config.py:171: error: Too many arguments for "items" of "Mapping"
Found 5 errors in 1 file (checked 1 source file)

---
### Try `RawConfigParse | None` ...
internetarchive/config.py:162: error: Need type annotation for "_config"
internetarchive/config.py:166: error: Incompatible return value type (got "Union[RawConfigParser, Dict[Any, Any]]", expected "Dict[Any, Any]")
internetarchive/config.py:169: error: Item "None" of "Optional[RawConfigParser]" has no attribute "sections"
internetarchive/config.py:171: error: Item "None" of "Optional[RawConfigParser]" has no attribute "items"
internetarchive/session.py:101: error: Argument 1 to "get_config" has incompatible type "Optional[Dict[Any, Any]]"; expected "Optional[RawConfigParser]"
Found 5 errors in 2 files (checked 1 source file)

### So I left is unspecified.